velox/expression/FunctionSignature.h

```diff
@@ -168,6 +168,16 @@ class FunctionSignature {
   }
 
  protected:
+  FunctionSignature(
+      std::unordered_map<std::string, SignatureVariable> variables,
```

Comment:
pass 'variables' as const &

Reply:
The constructor uses `std::move(variables)` to initialize `variables_` , therefore it cannot be const &.